src/main/groovy/com/palantir/launchconfig/EclipseLaunchConfigTask.groovy

```diff
@@ -89,6 +89,21 @@ class EclipseLaunchConfigTask extends DefaultTask {
         Files.write(launchFile, xmlString.getBytes("UTF-8")).toFile()
     }
 
+    /**
+     * Build the VM Params from the JVM Args and the System Properties.
+     *
+     * Note: this needs to be left as "protected" to avoid scoping issues above
+     */
+    protected List<String> buildVmParams(JavaExec javaExec) {
+        List<String> vmParams = new ArrayList<String>()
+        vmParams.addAll(javaExec.jvmArgs)
```

Comment:
shouldn't this be `allJvmArgs` to match the original implementation? (in both places)

Reply:
So in the original implementation Eclipse and Idea differed in their use of the `jvmArgs` api. Eclipse used `allJvmArgs` to pick up the system properties